<launch>
    <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_house.launch"/>
    <include file="$(find turtlebot3_slam)/launch/turtlebot3_slam.launch">
        <arg name="slam_methods" value="gmapping"/>
    </include>
    <node pkg="transformations_odom" name="tf_odom" type="poseconversions.py" output="log"/>
    <include file="$(find plan_path)/launch/costmap.launch"/>
    <include file="$(find tagstore)/launch/tagstore.launch"/>
</launch>
